scripts/ai2-internal/run_with_beaker.py

```diff
@@ -0,0 +1,80 @@
+#! /usr/bin/env python
+
+# Script to launch AllenNLP Beaker jobs.
+
+import argparse
+import os
+import subprocess
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(os.path.join(os.path.join(__file__, os.pardir), os.pardir))))
+
+from allennlp.commands.train import Train
+from allennlp.common.params import Params
+
+#TODO(michaels): add CLI support for mounting datasets.
+
+def main(param_file, description):
+    ecr_repository = "896129387501.dkr.ecr.us-west-2.amazonaws.com"
+    commit = subprocess.check_output(["git", "rev-parse", "HEAD"], universal_newlines=True).strip()
+    image = f"{ecr_repository}/allennlp/allennlp:{commit}"
+    overrides = ""
+
+    # Read params and set environment
+    params = Params.from_file(param_file, overrides)
+    flat_params = params.as_flat_dict()
+    env = []
+    for k, v in flat_params.items():
+        k = str(k).replace('.', '_')
+        env.append(f"--env={k}={v}")
+
+    # Get temporary ecr login. For this command to work, you need the python awscli
+    # package with a version more recent than 1.11.91.
```

Comment:
you could check this:

```
try:
    import awscli
    if awscli.__version__ <= ....
except ModuleNotFoundError:
    ...
```

probably more helpful than just leaving a comment

Reply:
It wasn't my comment (copied over) ;-)

But I'll take the suggestion.

Reply:
I'm not going to do this as it's more work than it's worth since the awscli version is a string with three parts.  Also, this package is in our requirements.txt.